docs/reference/github/workflow/branches.md

```diff
@@ -0,0 +1,148 @@
+# Developing on Branches
```

Comment:
I think this should be more of a Git-specific thing rather than putting it on the branches page. It seems a little out of place. We could consider adding our own Git reference in the future and add these sorts of things to it.

Reply:
Yea on second thought I think it is outside the scope of this site